src/hotspot/share/gc/shared/barrierSetRuntime.hpp

```diff
@@ -38,7 +38,9 @@ class BarrierSetRuntime: public AllStatic {
 public:
   // Template interpreter...
   static void value_copy(void* src, void* dst, InlineKlass* md);
+  static void value_copy2(void* src, void* dst, InlineLayoutInfo* layout_info);
```

Comment:
Are there not better names (e.g _ilo) than 2 ? Or is this temp naming ?

Reply:
This is a temporary naming, once flat arrays have been updated, we can remove the old value_copy() method, and reuse the name for the new method.